src/x/opentracing/tracing.go

```diff
@@ -73,6 +107,43 @@ func (cfg *TracingConfiguration) NewTracer(defaultServiceName string, scope tall
 	return tracer, jaegerCloser, nil
 }
 
+func (cfg *TracingConfiguration) newLightstepTracer(serviceName string) (opentracing.Tracer, io.Closer, error) {
+	if cfg.Lightstep.Tags == nil {
+		cfg.Lightstep.Tags = opentracing.Tags{}
+	}
+
+	tags := cfg.Lightstep.Tags
+	if _, ok := tags[lightstep.ComponentNameKey]; !ok {
+		tags[lightstep.ComponentNameKey] = serviceName
+	}
+
+	tags[spanTagBuildRevision] = instrument.Revision
+	tags[spanTagBuildBranch] = instrument.Branch
+	tags[spanTagBuildVersion] = instrument.Version
+	tags[spanTagBuildDate] = instrument.BuildDate
+	tags[spanTagBuildTimeUnix] = instrument.BuildTimeUnix
+	tags[spanTagBuildGoVersion] = runtime.Version()
```

Comment:
👍 these seem quite valuable - should we add them to `cfg.Jaeger.NewTracer(...)` with `cfg.Jaeger.NewTracer(..., jaeger.TracerOptions.Tag(key, value), jaeger.TracerOptions.Tag(key, value), ...)` as options?